ios-apps-master/Beer_Diary/Beer_Diary/Add_Beer.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  Add_Beer.swift
+//  Beer_Diary
+//
+//  Created by Piotr Mielcarzewicz on 13/11/16.
+//  Copyright © 2016 Piotr Mielcarzewicz. All rights reserved.
+//
+
+import UIKit
+
+struct BeerItem
+{
+    var name = ""
+    var brewery = ""
+    var style = ""
+    var country = ""
+    var thoughts = ""
+    var aroma = 0
+    var appearance = 0
+    var palate = 0
+    var taste = 0
+    var overall:Float = 0
+    var date = NSDate()
+}
+
+func saveData()
+{
+    var tname = [String]()
```

Comment:
Why is the name style `tsomething`? Apple usually recommends full words in variable names https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html